applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -1577,7 +1577,7 @@ void UPwSmallStrainElement<TDim, TNumNodes>::CalculateCauchyStrain(ElementVariab
 }
 
 template <unsigned int TDim, unsigned int TNumNodes>
-void UPwSmallStrainElement<TDim, TNumNodes>::CalculateCauchyGreenStrain(ElementVariables& rVariables)
+void UPwSmallStrainElement<TDim, TNumNodes>::CalculateGreenLagrangeStrain(ElementVariables& rVariables)
```

Comment:
Now I've got an idea on Cauchy and Lagrange strain tensors. Why was Cauchy here?

Reply:
I don't know, it was probably copied from another application